demo-plugin/console-extensions.json

```
@@ -0,0 +1,27 @@
+[
+    {
+        "type": "acm.application/action",
+        "properties": {
+            "id": "action1",
+            "title": "Test action",
+            "model": [
+                {
+                    "apiVersion": "app.k8s.io/v1beta1",
+                    "kind": "Application"
```

```suggestion
                    "apiVersion": "app.k8s.io/v1beta1",
                    "kind": "Application"
```
I wish the API supported group version kind. Just to unify it with console extensions. Maybe we can improve on it at a later time.